.github/actions/setJobEnv.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
```

Comment:
@wallentx if you're going to try and get this adopted among multiple repos, we should make it an action in https://github.com/Chia-Network/actions

Reply:
done!